### PR TITLE
W-17119871 OS: Alt Text: Review update Tracker

### PR DESCRIPTION
**Pull request by hannanelson**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ You can select Object Store v2 as the implementation for Mule 3 and Mule 4 in Cl
 
 See the xref:osv2-faq.adoc[FAQ] for Object Store information.
 
-image::osv2-architecture.png["Object Store v2 architecture"]
+image::osv2-architecture.png["A diagram illustrating how Mule and user-defined components connect to a central data store"]
```
Comment (hannanelson) [inline modules/ROOT/pages/index.adoc:9]: I'm deliberately being self-referential here.

File: modules/ROOT/pages/osv2-guide.adoc [hunk 2]
```diff
@@ -138,12 +142,12 @@ To configure a custom TTL period for keys stored in the object store:
 
 . Create an app that uses Mule runtime engine version 4.2.1 or later.
 . Configure the object store in the app.
-. Select the *Store* operation in the flow and, in the properties window, click the green plus icon (+) to the right of the *Object store* field:
+. Select the *Store* operation in the flow and, in the properties window, click the green plus icon (+) for the *Object store* field:
 +
-image::os-use-ttl-props.png["Green plus icon in the properties window"]
+image::os-use-ttl-props.png["Studio properties window for the Store operation showing the green plus icon"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/osv2-guide.adoc:147]: Can we use color in the descriptions?

File: modules/ROOT/pages/osv2-guide.adoc [hunk 3]
```diff
@@ -181,13 +185,13 @@ To create partitions in Object Store v2:
 . Configure the Store operation.
 . Configure the partitions:
 +
-.. In the properties window, click the green plus icon (+) to the right of the *Object store* field:
+.. In the properties window, click the green plus icon (+) for the *Object store* field:
 +
-image::os-use-ttl-props.png["Green plus icon in the properties window"]
-.. In the *Global Element Properties* window, click the green plus icon (+) to the right of *Configuration Reference* 
+image::os-use-ttl-props.png["Studio properties window for the Store operation showing the green plus icon"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/osv2-guide.adoc:190]: "green" okay?

File: modules/ROOT/pages/osv2-guide.adoc [hunk 4]
```diff
@@ -181,13 +185,13 @@ To create partitions in Object Store v2:
...
+.. In the *Global Element Properties* window, click the green plus icon (+) for *Configuration Reference* 
 to create a global object store configuration:
 +
-image::os-use-partition-studio7-os-global-props.png["Green plus icon for the Configuration Reference field in the Global Element Properties window"]
+image::os-use-partition-studio7-os-global-props.png["Global Element Properties window showing the green plus icon for the Configuration Reference field"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/osv2-guide.adoc:194]: Same as previous comments. I won't comment any more.

File: modules/ROOT/pages/osv2-tutorial.adoc [hunk 5]
```diff
@@ -45,34 +45,39 @@ To configure a Mule 4.x app in Studio 7.x:
...
+image::os-tutorial-studio7-palette.png["Mule Palette view showing the HTTP Connector and Listener operations"]
++
+[calloutlist]
+.. HTTP Connector
+.. Listener operation
+
 . Click *ObjectStore* and then drag the *Store* operation to *Listener* on the canvas:
 +
-image::os-tutorial-studio7-flow.png["Store operation in the Studio 7 canvas"]
+image::os-tutorial-studio7-flow.png["Studio 6.x window showing the Store operation on the canvas"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/osv2-tutorial.adoc]: Isn't Studio 6 deprecated?

File: modules/ROOT/pages/osv2-tutorial.adoc [hunk 6]
```diff
@@ -125,7 +130,7 @@ To test your app:
 . Click the *Status* column for the `osv2-demo` app to display its details.
 . Copy the *App url* value:
 +
-image::os-tutorial-rtm-deploy-url.png["App url in Runtime Manager"]
+image::os-tutorial-rtm-deploy-url.png["Runtime Manager app details pane showing the App url"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/osv2-tutorial.adoc:133]:
```suggestion
image::os-tutorial-rtm-deploy-url.png["Runtime Manager app details pane showing the App URL"]
```

File: modules/ROOT/pages/osv2-tutorial.adoc [hunk 7]
```diff
@@ -270,31 +275,31 @@ include::partial$osv2-faq.adoc[tag=faqKeyChars]
...
-image::os-tutorial-studio6-os-global-props.png["Partition and Object Store Reference fields in the Global Element Properties window"]
+image::os-tutorial-studio6-os-global-props.png["Global Element Properties window showing the Partition and Object Store Reference fields"]
 +
 [NOTE]
 The value of the *Object Store Reference* field must be `_defaultUserObjectStore` for Object Store v2 to store data.
 +
 The *ObjectStore* properties window now indicates that there are no errors:
 +
-image::os-tutorial-studio6-os-props.png["ObjectStore properties in Studio 6"]
+image::os-tutorial-studio6-os-props.png["ObjectStore properties window showing no errors"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/osv2-tutorial.adoc:285]: Object Store?

File: modules/ROOT/pages/osv2-tutorial.adoc [hunk 8]
```diff
@@ -315,7 +320,7 @@ To test your app:
 . Click the *Status* column for the `osv2-demo-studio-6` app to display its details.
 . Copy the *App url* value:
 +
-image::os-tutorial-rtm-deploy-url-3.png["App url in Runtime Manager"]
+image::os-tutorial-rtm-deploy-url-3.png["Runtime Manager app details pane showing the App url"]
```
Comment (valkyrie69) [inline modules/ROOT/pages/osv2-tutorial.adoc:323]:
```suggestion
image::os-tutorial-rtm-deploy-url-3.png["Runtime Manager app details pane showing the App URL"]
```

**Review by valkyrie69 — DISMISSED** [review]

File: modules/ROOT/pages/osv2-tutorial.adoc (hunk 5 above)
Comment (hannanelson) [reply modules/ROOT/pages/osv2-tutorial.adoc]: Studio 6.x is still available, but now I'm wondering if that image is correct 🤣

File: modules/ROOT/pages/osv2-tutorial.adoc (hunk 6 above)
Comment (hannanelson) [reply modules/ROOT/pages/osv2-tutorial.adoc:133]: The label on the UI element is "Url". For better or for worse.

File: modules/ROOT/pages/osv2-tutorial.adoc (hunk 7 above)
Comment (hannanelson) [reply modules/ROOT/pages/osv2-tutorial.adoc:285]: The label in the UI is CamelCase!

File: modules/ROOT/pages/osv2-guide.adoc (hunk 2 above)
Comment (hannanelson) [reply modules/ROOT/pages/osv2-guide.adoc:147]: Yes.

File: modules/ROOT/pages/osv2-tutorial.adoc (hunk 5 above)
Comment (hannanelson) [reply modules/ROOT/pages/osv2-tutorial.adoc]: Fixed.

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**